### DISCUSSION TITLE
Add deployment checklist

### DISCUSSION THREAD

**Discussion by sharifsalah** [google/docsy#1532] [Ideas]
There’s a lot of moving parts in Docsy / Hugo. If this is the first time someone is setting up a standalone site, there’s a lot to think about, such as linking to a privacy policy, search config, accessibility, and handling contributions.

Consider adding a deployment checklist to allow the site manager to run through and make sure they’ve configured and tested everything they should be thinking about before they share the new site with the world.

**Comment by emckean** [comment]
This is a great idea, maybe we can crowdsource this, especially now that a new release of Docsy is imminent?
